lib/decidim/questions/component.rb

```diff
@@ -163,6 +163,33 @@
       global = nil
     end
 
+    email = "committee-user-#{participatory_space.underscored_name}-#{participatory_space.id}@example.org"
+    name = "#{Faker::Name.name} #{participatory_space.id} #{n} committee"
+
+    committee_user = Decidim::User.find_or_initialize_by(email: email)
+    author.update!(
```

Comment:
Shouldn't it be "committee_user"?
```suggestion
    committee_user.update!(
```